modules/docs/mdx/9.0-UPGRADE-GUIDE.mdx

```diff
@@ -81,6 +82,14 @@ you use the Preview version of `SidePanel` until the one in Main is replaced wit
 
 ---
 
+### Stack, HStack, VStack
+
+We've [hard-deprecated](#hard-deprecation) `Stack`, `HStack` and `VStack`. These components have
+been fully removed from Canvas Kit. Consider using the
+[Flex](https://workday.github.io/canvas-kit/?path=/docs/components-containers-flex--basic) component.
```

Comment:
I wonder if the codemod from 8.4 should be part of the transform to handle this

Reply:
Yeah this will be added into the v9 codemod